fsm/eventprocessor.go

```diff
@@ -144,6 +146,15 @@ func (em eventProcessor) Apply(evt statemachine.Event, user interface{}) (EventN
 	return e.name, completeEvent(e, nil)
 }
 
+func (em eventProcessor) ClearEvents(evts []statemachine.Event, err error) {
```

Comment:
`err` is required here?  If so, we should assert or something on it?

Reply:
see completeEvent call